ios/RiveReactNativeView.swift

```diff
@@ -41,6 +41,8 @@ class RiveReactNativeView: RCTView, RivePlayerDelegate, RiveStateMachineDelegate
     
     @objc var fit: String?
     
+    @objc var layoutScaleFactor: NSNumber = -1.0 // -1.0 will inform the iOS runtime to determine the correct scale factor automatically
```

Comment:
This type is a `Double` in Swift (and likely ObjC by interop). What's the decision behind using NSNumber?

Reply:
Double doesn't work for nullable(optional) values in these bindings: https://reactnative.dev/docs/legacy/native-modules-ios#argument-types